var/spack/repos/builtin/packages/spglib/package.py

```diff
@@ -37,6 +45,17 @@ class Spglib(CMakePackage):
     version("1.10.1", sha256="8ed979cda82f6d440567197ec191bffcb82ee83c5bfe8a484c5a008dd00273f0")
     version("1.10.0", sha256="117fff308731784bea2ddaf3d076f0ecbf3981b31ea1c1bfd5ce4f057a5325b1")
 
+    variant("openmp", default=True, when="@1.16.2:")
+
     @property
     def libs(self):
         return find_libraries("libsymspg", root=self.prefix, shared=True, recursive=True)
+
+    def cmake_args(self):
+        return [self.define_from_variant("USE_OMP", "openmp")]
+
+    def url_for_version(self, version):
+        if version <= Version("1.16.1"):
+            return f"https://github.com/atztogo/spglib/archive/v{version}.tar.gz"
+        else:
+            return f"https://github.com/spglib/spglib/archive/refs/tags/v{version}.tar.gz"
```

Comment:
The older packages are no longer available at that URL so cannot be checksummed.

It would be better to restore the `url` property, optionally updating it to the latest version if desired, since all of the listed versions are available there and with the same checksums shown here.

Reply:
Thanks @tldahlgren. I did not check older versions with `spack checksum` (and can confirm that they don't work), but I tried to install a couple of them and it worked (`spack install spglib@1.10.0`). Apparently it was fetching from a mirror. I'll change it as you suggest.